osu.Game.Rulesets.Tau/UI/Cursor/AbsoluteCursor.cs

```diff
@@ -15,24 +15,25 @@ public AbsoluteCursor()
 
             Children = new Drawable[]
             {
-                new CircularProgress
-                {
-                    RelativeSizeAxes = Axes.Both,
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                    Current = new BindableDouble(.33f),
-                    InnerRadius = 0.1f,
-                    Rotation = -150
-                },
-                new CircularProgress
-                {
-                    RelativeSizeAxes = Axes.Both,
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                    Current = new BindableDouble(.33f),
-                    InnerRadius = 0.1f,
-                    Rotation = 30
-                },
+                // new CircularProgress
+                // {
+                //     RelativeSizeAxes = Axes.Both,
+                //     Anchor = Anchor.Centre,
+                //     Origin = Anchor.Centre,
+                //     Current = new BindableDouble(.33f),
+                //     InnerRadius = 0.1f,
+                //     Rotation = -150
+                // },
+                // new CircularProgress
+                // {
+                //     RelativeSizeAxes = Axes.Both,
+                //     Anchor = Anchor.Centre,
+                //     Origin = Anchor.Centre,
+                //     Current = new BindableDouble(.33f),
+                //     InnerRadius = 0.1f,
+                //     Rotation = 30
+                // },
+                new SkinnableDrawable(new TauSkinComponent(TauSkinComponents.Cursor), _ => new CursorPiece(), ConfineMode.ScaleToFit)
```

Comment:
What's the size are we trying to aim here? Currently if the user provides a texture any larger than `40x40`, it'll scale accordingly.

Reply:
if it's that, then any multiple of 40x40 is good enough

Reply:
No what i'm saying is that any resolution would work with this.

Reply:
of course, in that case, then it's only a 1:1 png, nothing that big of a deal.